iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -1127,6 +1127,11 @@ static unsigned decideFusableLinalgOps(mlir::FuncOp funcOp) {
                 consumerIndexingMap.getResults()) {
           continue;
         }
+        if (llvm::any_of(
```

Comment:
WHy this?

Reply:
I have to see some examples here. These heuristics have become really long and patch-worky. Need to resolve this and do some house cleaning here.

Ill stamp this for now.